stylesheets/_component.notifications.scss

```diff
@@ -0,0 +1,99 @@
+.notifications {
+    float: right;
+    position: static; // override default dropdown behaviour for small screens
+
+    @include respond-min($screen-phone) {
+        position: relative; // restore default dropdown behaviour
+    }
+}
+
+.notifications-toggle {
+    color: color(text, light);
+    float: right;
+    font-size: 24px;
+    padding: 4px 0;
+    text-decoration: none;
+
+    @include respond-min($screen-phone) {
+        padding: 10px 0 0 0;
+    }
+
+    &.is-active {
+        color: color(black);
+
+        // the red dot indicator
+        &:before {
+            background-color: color(danger);
+            border: 2px solid color(white);
+            border-radius: 10px;
+            content: '';
+            display: block;
+            position: absolute;
+            right: -2px;
+            top: 3px;
+            height: 10px;
+            width: 10px;
+        }
+    }
+}
+
+.notifications-list {
+    border: 1px solid color(border);
+    clear: both;
+    left: 10px;
+    padding: 0; // reset .dropdown__menu defaults
+    position: absolute;
+    right: 10px;
+    top: 60px;
+    width: auto;
+
+    @include respond-min($screen-phone) {
+        left: auto;
+        right: -180px;
+        top: 40px;
+        width: 400px;
+    }
+}
+
+.notifications__header {
+    border-bottom: 1px solid color(border);
+    float: left;
+    padding: 10px;
+    width: 100%;
+}
+
+.notifications-title {
+    display: inline-block;
+    line-height: 1.2em;
```

Comment:
Do you need the `em` for these line-heights or could you use a unitless ones instead?